README.md

```diff
@@ -1,60 +1,82 @@
 # cuCollections
-## Table of Content
-- [Building cuCollections](#building-cuCollections)
-- [Quick Start](#quick-start)
-- [Integration](#integration)
-- [References](#references)
 
-## Building cuCollections
-Building cuCollections is very straightforward. If your system doesn't have CUDA Toolkit, please install it. The latest version of CUDA toolkit can be found [here](https://developer.nvidia.com/cuda-downloads). Subsequently, please perform the following steps to build the project.
-
-**Prerequisite**: Make sure CUDA Toolkit is installed on your system.
-```bash
-export CUDACXX=/path/to/nvcc
-cuCollectionsPath=$(pwd)/cuCollections
-git clone --recurse-submodules  https://github.com/rapidsai/cuCollections.git $cuCollectionsPath
-cd $cuCollectionsPath
-git submodule update --init --recursive
-mkdir build && cd build
-cmake ..
-make -j$(nproc)
-make test # optional
+`cuCollections` (`cuco`) is an open-source, header-only library of GPU-accelerated, concurrent data structures. 
+
+Similar to how [Thrust](https://github.com/thrust/thrust) and [CUB](https://github.com/thrust/cub) provide STL-like, GPU accelerated algorithms and primitives, `cuCollections` provides STL-like concurrent data structures. `cuCollections` is not a one-to-one, drop-in replacement for STL data structures like `std::unordered_map`. Instead, it provides functionally similar data structures tailored for efficient use with GPUs. 
+
+## Development Status
+
+`cuCollections` is still under heavy development. Users should expect breaking changes and refactoring to be common. 
+
+## Getting cuCollections
+
+`cuCollections` is header only and can be incorporated manually into your project by downloading the headers and placing them into your source tree.
+
+### Adding `cuCollections` to a CMake Project
+
+`cuCollections` is designed to make it easy to include within another CMake project. The `CMakeLists.txt` exports a `cuco` target that can be linked[1] into a target to setup include directories, dependencies, and compile flags necessary to use `cuCollections` in your project. 
```

Comment:
can we make `[1]` a hyperlink that users can click on and get to the footnote? it was not straightforward to me if it's a typo or some reference.